app/client/public/index.html

```diff
@@ -67,6 +67,16 @@
 			gtag("config", "UA-92076314-27");
 		</script>
 
+		<!-- Google tag (gtag.js) -->
```

Comment:
There's already a tag existing do we need both of them or can we remove the old one? Also the ci build is failing.

Reply:
Yes, we have have the dual support for this. Once the older is depreciated, we can remove it.

Reply:
Looking in the build issue. Thanks @gdsoumya